src/librustdoc/clean/inline.rs

```diff
@@ -290,10 +290,12 @@ pub fn build_ty(cx: &DocContext<'_>, did: DefId) -> Option<clean::Type> {
     }
 }
 
+// Passed `DefId` has to be an an ADT (struct/union/enum) or Trait item/path/reexport.
```

Comment:
Is it supposed to be just a comment or a doc comment?

Reply:
make them doc comments.